justin-tan/week4/thurs/crud-hw/views/index.erb

```diff
@@ -0,0 +1,11 @@
+
+
+<% @results.each do |el| %>
+
+<a href="/profiles/<%= el['id'] %>">
+<img src="<%= el['profile_img'] %>" alt="<%= el['name'] %>">
```

Comment:
probably give this a class and set the height in css. they are all full screen images hahaha tooo big